.github/workflows/native_s3_dlr.yml

```diff
@@ -0,0 +1,38 @@
+name: Native S3 DLR
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [macos-latest]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Clone DLR
```

Comment:
You can use git checkout plugin in here. Refer to the TFLite build

Reply:
right but I can't find a way to pass the env back to ref, seems like ref has to be a constant